### ISSUE TITLE
what is the property file in dataset? I just have .idx and .rec files, how to create property file?

### ISSUE THREAD

**Issue by hdjsjyl** [deepinsight/insightface#156]
what is the property file in dataset? I just have .idx and .rec files, how to create property file? Thanks.

**Comment by ahong007007** [comment]
我根据mxnet生成.lst和.rec文件，但是在使用CUDA_VISIBLE_DEVICES='0,1,2,3' python -u train_softmax.py --network m1 --loss-type 12 --lr 0.005 --mom 0.0 --per-batch-size 150 --data-dir ../datasets/faces_ms1m_112x112 --pretrained ../model-m1-softmax,50 --prefix ../model-m1-triplet
finetuning时，提示没有property文件，是这样的错误吗，谢谢
